apply_extra.bash

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# global variables
+FLATPAK_ID='__FLATPAK_ID__'
```

Comment:
this is in the environment by default. It this necessary.

Reply:
Apparently not for `apply_extra`.

Reply:
not relevant anymore (code changed)